docs/api.md

```diff
@@ -20,19 +20,23 @@ Following is the supported API format for prometheus encode:
                  filter: an optional criterion to filter entries by. Deprecated: use filters instead.
                      key: the key to match and filter by
                      value: the value to match and filter by
-                     type: (enum) the type of filter match: exact (default), presence, absence or regex
-                         exact: match exactly the provided fitler value
+                     type: (enum) the type of filter match: exact (default), exact_not, presence, absence, regex or regex_not
+                         exact: match exactly the provided filter value
+                         exact_not: the value must be different from the provided filter
```

Comment:
`different` ?

Reply:
or maybe we can rename both to `equal` / `not_equal`

That will allow to extrapolate on any cases such as `equal_ignore_case`...

Reply:
I'm fine with this renaming, `equal`/`not_equal` sounds more familiar